components/gc-most-requested/index.json-ld

```diff
@@ -7,28 +7,175 @@
 		"modified": "dct:modified"
 	},
 	"title": {
-		"en": "Most requested - Documentation and working example",
-		"fr": "En demande - Documentation et example pratique"
+		"en": "Most requested",
+		"fr": "En demande"
 	},
 	"description": {
 		"en": "Features top tasks related to the page it is on.",
 		"fr": "Comprend des tâches principales liées à la page sur laquelle elles se trouvent."
 	},
-	"modified": "2021-05-20",
+	"modified": "2024-02-14",
 	"componentName": "gc-most-requested",
-	"status": "provisional",
+	"status": "stable",
+	"version": "1.0",
 	"pages": {
+		"docs": [
+			{
+				"title": "Most requested - Documentation",
+				"language": "en",
+				"path": "gc-most-requested-doc-en.html"
+			},
+			{
+				"title": "En demande - Documentation",
+				"language": "fr",
+				"path": "gc-most-requested-doc-fr.html"
+			}
+		],
 		"examples": [
 			{
-				"title": "Most requested - Documentation and working example",
+				"title": "Most requested",
 				"language": "en",
 				"path": "gc-most-requested-en.html"
 			},
-		{
+			{
 				"title": "En demande",
 				"language": "fr",
-				"path": "gc-most-requested-fr.html - Documentation et example pratique"
+				"path": "gc-most-requested-fr.html"
+			},
+			{
+				"title": "Most requested - bad implementation",
+				"language": "en",
+				"path": "gc-most-requested-bad-en.html"
+			},
+			{
+				"title": "En demande - mauvaise implémentation",
+				"language": "fr",
+				"path": "gc-most-requested-bad-fr.html"
 			}
 		]
-	}
+	},
+	"dependencies": "No dependency.",
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Most requested - default",
+				"fr": "En demande - par défaut"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Features top tasks related to the page it is on.",
+				"fr": "Comprend des tâches principales liées à la page sur laquelle elles se trouvent."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/common-design-patterns/most-requested.html",
+				"fr": "https://conception.canada.ca/configurations-conception-communes/en-demande.html"
+			},
+			"iteration": "_:iteration_mostrequested_1",
+			"example": [
+				{
+					"en": { "href": "gc-most-requested-en.html", "text": "Most requested" },
+					"fr": { "href": "gc-most-requested-fr.html", "text": "En demande" }
+				},
+				{
+					"en": { "href": "gc-most-requested-bad-en.html", "text": "Most requested - bad implementation" },
+					"fr": { "href": "gc-most-requested-bad-fr.html", "text": "En demande - mauvaise implémentation" }
+				}
+			],
+			"implementation": [
+				"_:implement_mostrequested"
+			],
+			"history": [
+				{
+					"en": "February 2024 - Initial implementation of the variation.",
+					"fr": "Février 2024 - Implémentation initiale de la variante."
```

Comment:
J'indiquerais lorsqu'on l'a introduit en tant que provisional feature

```suggestion
					"en": [
						"February 2024 - Initial and stable implementation of the variation.",
						"June 2021 - Provisional implementation"
					],
					"fr": [
						"Février 2024 - Stabilisation et implémentation initiale de la variante.",
						"Juin 2021 - Implémentation provisionnelle"
					]
```

Idéalement, il y aurait une toute petite itération (id+name seulement) pour représenter lorsqu'il était provisionnelle et l'itération actuelle aurait une relation avec celle qui la précède (predecessor).